### PR TITLE
Hack 13 - feat: Create a back button with href javascript

### PR DESCRIPTION
**Pull request by brianndiiri**

### CODE REVIEW

File: webflow-hacks/misc-hacks/13/13.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// when the DOM is ready
+document.addEventListener('DOMContentLoaded', () => {
+  const BUTTON_JS_SELECTOR = '[fs-hacks-element="hack13-back-1"]';
+  const BUTTON_JQUERY_SELECTOR = '[fs-hacks-element="hack13-back-2"]';
```
Comment (alexiglesias93) [inline webflow-hacks/misc-hacks/13/13.ts]: I don't understand why are the selectors called JS and JQUERY

Comment (brianndiiri) [reply webflow-hacks/misc-hacks/13/13.ts]: sorry bad autocomplete. Fixing....

File: webflow-hacks/misc-hacks/13/13.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+// when the DOM is ready
+document.addEventListener('DOMContentLoaded', () => {
+  const BACK_ONE_SELECTOR = '[fs-hacks-element="hack13-back-1"]';
+  const BACK_TWO_SELECTOR = '[fs-hacks-element="hack13-back-2"]';
+
+  // on click
+  const backOne = document.querySelector<HTMLDivElement>(BACK_ONE_SELECTOR);
+  backOne?.addEventListener('click', function (e) {
```
Comment (alexiglesias93) [inline webflow-hacks/misc-hacks/13/13.ts]: Remember to not use optional chaining when possible in the Hacks Rewrite

**Review by alexiglesias93 — CHANGES_REQUESTED** [review]

File: webflow-hacks/misc-hacks/13/13.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
+// when the DOM is ready
+document.addEventListener('DOMContentLoaded', () => {
+  const BACK_ONE_SELECTOR = '[fs-hacks-element="hack13-back-1"]';
+  const BACK_TWO_SELECTOR = '[fs-hacks-element="hack13-back-2"]';
+
+  // on click
+  const backOne = document.querySelector<HTMLDivElement>(BACK_ONE_SELECTOR);
+  const backTwo = document.querySelector<HTMLDivElement>(BACK_TWO_SELECTOR);
+  if (!backOne || !backTwo) return;
```
Comment (alexiglesias93) [inline webflow-hacks/misc-hacks/13/13.ts:7]: Now this is assuming that users need to have both backOne and backTwo buttons on the page.
But users will probably will just implement either one or the other, but not both, right?

Can we split this in to different functions, so users can choose from?

**Review by alexiglesias93 — CHANGES_REQUESTED** [review]

File: webflow-hacks/misc-hacks/13/13.ts (hunk 3 above)
Comment (brianndiiri) [reply webflow-hacks/misc-hacks/13/13.ts:7]: So two different hacks @alexiglesias93?

Comment (alexiglesias93) [reply webflow-hacks/misc-hacks/13/13.ts:7]: Let's keep it like this, two hacks will confuse users more

**Review by alexiglesias93 — APPROVED** [review]

**Pull request merged**